lib/src/particle/particle.dart

```diff
@@ -32,7 +32,7 @@ class Particle {
   /// Use this to store application-specific body data.
   Object? userData;
 
-  Particle(this.system, ParticleGroup? group) {
+  Particle(this.system, [ParticleGroup? group]) {
```

Comment:
Not sure I understand what you mean, what is not consistent? Named optional parameters are used a lot more than positional ones in forge2d I think.

Reply:
Just saw that World also had positional parameters too, I think I like named ones better, but there is no recommendation in the style guides from what I can find.

Reply:
I think that since we use named parameters in Flame we'd like to keep it the same here. I think that is the last thing to fix.
So all positional ones should be turned into named ones, if there are more positional ones.
And all the todo comments that you put out can be removed, and then we can merge!